tasks/e2e.sh

```diff
@@ -135,7 +138,7 @@ npm install $cli_path
 
 # Install the app in a temporary location
 cd $temp_app_path
-create_react_app --scripts-version=$scripts_path test-app
+create_react_app --scripts-version=$scripts_path --template=$root_path/packages/react-scripts/fixtures/kitchensink test-app
```

Comment:
No, just that we should have both of them. It's fine to run both but I want to run the simple one first to catch easy issues. It will be trickier to debug kitchensink.